stack_elk/src/ls_telemetry.conf

```diff
@@ -30,6 +30,8 @@ input {
 		'mplstesummary', 'RootOper~MPLS_TE~Tunnels~Summary',
 		'mplstetopology', 'RootOper~MPLS_TE~Topology',
 		'mplsteautoroute', 'RootOper~MPLS_TE~AnnounceTunnelsInfo~AutorouteAnnounceTable~(?<RouterID>.*)~(?<Protocol>.*)~(?<Area>\d+)~(?<Instance>.*)']
+	      #
```

Comment:
The xform_flat_keys also need to be updated, to include the new version variant. The path is slightly different now.

Example:

we will also need to show a few matches for the new format:
'countersPre601', 'RootOper~InfraStatistics~(?<InterfaceName>.*)~Latest~GenericCounters',
'counters': 'RootOper~InfraStatistics~Interface~(?<InterfaceName>.*)~Latest~GenericCounters',
'interfacesPre601', 'RootOper~Interfaces~(?<InterfaceName>.*)',
'interfaces', 'RootOper~Interfaces~Interface~(?<InterfaceName>.*)',